src/index.ts

```diff
@@ -249,7 +249,7 @@ export const ics = (calendarEvent: CalendarEvent): string => {
     },
     {
       key: "UID",
-      value: Math.floor(Math.random() * 100000).toString().replace(".", ""),
+      value: event.uid || Math.floor(Math.random() * 100000).toString().replace(".", ""),
```

Comment:
My concern here is that you rewrote the tests to always use UID, this means that all test coverage has been lost for this `Math.floor` section. We should have at least one test that confirms that UIDs are generated as expected when a user **does not** provide a UID.

Reply:
Sounds good, added a new wrapper to test id for ics calls in the tests.